Assets/MRTK/Core/Utilities/Gltf/Schema/GltfAccessor.cs

```diff
@@ -31,7 +32,11 @@ public class GltfAccessor : GltfChildOfRootProperty
         /// 5125 (UNSIGNED_INT) is only allowed when the accessor contains indices
         /// i.e., the accessor is only referenced by `primitive.indices`.
         /// </summary>
-        public GltfComponentType componentType;
+        [NonSerialized]
+        public GltfComponentType ComponentType;
```

Comment:
Can we make this a property, to try to reduce the number of `public` fields we have and so we won't need the `NonSerialized` tag?